spark/pom.xml

```diff
@@ -0,0 +1,344 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- ~ Copyright 2016 Crown Copyright ~ ~ Licensed under the Apache License, 
+	Version 2.0 (the "License"); ~ you may not use this file except in compliance 
+	with the License. ~ You may obtain a copy of the License at ~ ~ http://www.apache.org/licenses/LICENSE-2.0 
+	~ ~ Unless required by applicable law or agreed to in writing, software ~ 
+	distributed under the License is distributed on an "AS IS" BASIS, ~ WITHOUT 
+	WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. ~ See the 
+	License for the specific language governing permissions and ~ limitations 
+	under the License. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>gaffer2</artifactId>
+		<groupId>gaffer</groupId>
+		<version>0.3.6-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>spark</artifactId>
+
+	<properties>
+		<maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
Do we need maven.compiler.source and maven.compiler.target properties? The gaffer parent pom sets the source and target java versions to 1.7 in the build plugin management section.